### PR TITLE
Header final

### PR DESCRIPTION
**Pull request by ochanje210**
this is the final header component PR!!!!

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -49,7 +48,7 @@ gulp.task('build:test-json', ['clean:test'], () => gulp.src('src/**/*.json')
   .pipe(gulp.dest('./dist-test/'))
 );
 
-gulp.task('build:test', ['build:test-client', 'build:test-server', 'build:test-json']);
+gulp.task('build:test', ['build:test-server', 'build:test-json']);
```
Comment (fguy) [inline gulpfile.babel.js]: Why did you get rid of build:test-client?

Comment (ochanje210) [reply gulpfile.babel.js]: hmm weird it was fixed before. I will fix it again

**Comment by ovekyc** [comment]
LGTM

**Comment by fguy** [comment]
shipit

**Comment by fguy** [comment]
Shipit

**Comment by fguy** [comment]
LG2M

**Pull request merged**